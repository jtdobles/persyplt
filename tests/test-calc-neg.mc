void main(){
	int a = 8 + -2;
	printi(a);
	float b = 9.7 + -0.4;
	printf(b);
}