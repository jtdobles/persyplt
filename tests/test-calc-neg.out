6
9.300000
